handlers/validator.go

```diff
@@ -1811,10 +1827,15 @@ func ValidatorSync(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	//remove all future sync periods
 	latestEpoch := services.LatestEpoch()
-	for syncPeriods[0].EndEpoch > services.LatestEpoch() {
-		syncPeriods = syncPeriods[1:]
+
+	//remove scheduled committees
+	for _, syncPeriod := range syncPeriods {
```

Comment:
Because of this change, the block starting at line 1842 is not required anymore, is it?

Reply:
It is required, can give you explanation after lunch lol

Reply:
So basically the block starting at line 1842 does a completely different thing. `StartEpoch` in that code block refers to the epoch the sync table first shows, which per default is that sync periods chronologically **latest** epoch due to the descending order. If that sync periods latest epoch has yet to happen, the `StartEpoch` is set to the current epoch, so that the current epoch is shown on the first page in the table, i.e. the table "starts" with that epoch.

Reply:
This is also the reason why I use the terminology `FirstEpoch` and `LastEpoch` in the validator sync participation, rather than `StartEpoch` and `EndEpoch`. Depending on the ordering of the sync table, `StartEpoch` could be the first or the last epoch.